### DISCUSSION TITLE
How to recognize USB drive?

### DISCUSSION THREAD

**Discussion by naizena-naiz** [waydroid/waydroid#1524] [Q&A]
I have a usb plugged in and waydroid can't recognize it. I cannot change USB preferences, there is no way to do it ( it appears in gray color). Any suggestion? Thanks in advance.

Volla X running Ubuntu Touch 20.04 OTA-5

**Comment by Michel49460** [comment]
Bonjour,
J'ai installé Waydroid dans Ubuntu mais Waydroid ne voit pas une clé usb. Quelle est la modification a apporté.
Merci par avance
Michel

**Comment by naizena-naiz** [comment]
Tengo Waydroid en Ubuntu Touch, pero tampoco reconoce los dispositivos USB. Parece ser que hay una forma de transferir archivos desde Ubuntu a Waydroid, pero no lo he probado. Pero el USB no lo reconoce. Siento no poder ayudarte.



El Sat, 8 Feb 2025 07:47:29 -0800, Michel49460 escribió:
> Bonjour,
> J'ai installé Waydroid dans Ubuntu mais Waydroid ne voit pas 
> une clé usb. Quelle est la modification a apporté.
> Merci par avance
> Michel
>

-- 
Sent with Dekko - the convergent email app
